chain/types/ethtypes/eth_types.go

```diff
@@ -203,14 +203,26 @@ func init() {
 	}
 }
 
+func NewEmptyEthBloom() []byte {
+	b := make([]byte, len(EmptyEthBloom))
+	copy(b, EmptyEthBloom[:])
+	return b
```

Comment:
How about just this, and get rid of `EmptyEthBloom` entirely; there's no need to copy nothing and it's never touched anywhere else.

```suggestion
	return [EthBloomSize / 8]byte{}
```